docs/user-guide/leverage-cli/reference/terraform.md

```diff
@@ -171,6 +171,10 @@ leverage terraform validate-layout
 
 Check the Terraform backend configuration in the code definition.
 
+
+!!! info "When you first create the backend layer for the first time, the bucket does not exist yet. If the Leverage CLI is used with validations, it will find that the Key does not exist or cannot be created. Thus, there's a need to first create the bucket and then move the "tfstate" file to it using the mentioned flag.""
```

Comment:
Just adding another possible phrasing:
> When you create the backend layer for the very first time, the S3 bucket does not exist yet. When running validations, Leverage CLI will find that the S3 Key does not exist or cannot be created. Thus, there's a need to first create the bucket, using the `init`'s `--skip-validation` flag, and then move the "tfstate" file to it.

I add the name of the flag here because, although we can reach it from a link in the `init` command, if we reach this section by scrolling down we do not have the context to implicitly talk about the mentioned flag.

Reply:
@juanmatias Ok I agree with you. May I propose to rephrase it a little bit?, what do you think about this?

> When you are setting up the backend layer for the very first time, the S3 bucket does not yet exist. When running validations, Leverage CLI will detect that the S3 Key does not exist or cannot be generated. Therefore, it is necessary to first create the S3 bucket by using the init `--skip-validation` flag in the initialization process, and then move the "tfstate" file to it.

Reply:
It's ok for me.